bal_schema_v1.5.json

```
@@ -0,0 +1,205 @@
+{
+    "$schema": "https://specs.frictionlessdata.io/schemas/table-schema.json",
+    "name": "base_adresse_locale",
+    "title": "Base Adresse Locale",
+    "description": "Spécification du fichier d'échange visant à diffuser des données voies-adresses locales",
```

- suggestion de rajouter la version de la spec à laquelle correspond le schema dans le "title" et la "description". 
```suggestion
    "title": "Base Adresse Locale v1.5",
    "description": "Spécification du fichier d'échange visant à diffuser des données voies-adresses locales v1.5",
```